sqlmesh/cli/main.py

```diff
@@ -891,18 +891,25 @@ def create_external_models(obj: Context, **kwargs: t.Any) -> None:
     type=str,
     help="Schema used for temporary tables. It can be `CATALOG.SCHEMA` or `SCHEMA`. Default: `sqlmesh_temp`",
 )
+@click.option(
+    "--select-model",
```

Comment:
```suggestion
    "--select-model",
    "-s",
```
makes the flag look less intimidating to reach for

Reply:
nit: Can we just shorten it to 

```
--model
-m
```

`m` Feels more intuitive than an `s`?

Reply:
Then we'll have to change it for all other similar flags. I disagree for this PR.

Reply:
I agree with Sung since we have the selector in the other commands as well so we can't change `--select-model`. though for the shorten version I opted for `-m` since the `-s` flag is already used for `--skip-columns`